cypress/integration/plugins/alerting-dashboards-plugin/monitors_dashboard_spec.js

```diff
@@ -107,8 +107,13 @@ describe('Monitors dashboard page', () => {
   });
 
   it('Displays expected number of alerts', () => {
+    // Wait for table to finish loading
+    cy.get('tbody > tr').should(($tr) =>
+      expect($tr).to.have.length.greaterThan(1)
+    );
+
     // Ensure the 'Monitor name' column is sorted in ascending order by sorting another column first
-    cy.contains('Last updated by').click({ force: true });
+    cy.contains('Last notification time').click({ force: true });
```

Comment:
For context, the `Last updated by` column was removed from this UI in PR https://github.com/opensearch-project/alerting-dashboards-plugin/pull/767